bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<filename>\S+):(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL})
+class HamlLintBear:
+    """
+    Checks code using ``haml-lint`` for
+    for HAML-specific style and lint checks
+    to ensure clean and readable HAML code
+    """
+    LANGUAGES = {'Haml'}
+    REQUIREMENTS = {GemRequirement('haml_lint', '0.26.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    SEE_MORE = 'https://github.com/brigade/haml-lint'
+
+    def create_arguments(self, filename, file, config_file):
+        args = ('--no-summary', filename)
+        if config_file:
```

Comment:
Also, to make the CI pass, add a setting by supplying a custom `hamllint_config`. You can refer to the tests of other bears.